var/spack/repos/builtin/packages/aocl-da/package.py

```diff
@@ -0,0 +1,123 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os
+
+from spack.package import *
+from spack.util.environment import EnvironmentModifications
+
+
+class AoclDa(CMakePackage):
+    """
+    The AOCL Data Analytics Library (AOCL-DA) is a data analytics library
+    providing optimized building blocks for data analysis. It is written with a
+    C-compatible interface to make it as seamless as possible to integrate
+    with the library from whichever programming language you are using.
+    The intended workflow for using the library is as follows:
+    • load data from memory by reading CSV files or using the in-built
+    da_datastore object
+    • preprocess the data by removing missing values, standardizing, and
+    selecting certain subsets of the data, before extracting contiguous
+    arrays of data from the da_datastore objects
+    • data processing (e.g. principal component analysis, linear model
+    fitting, etc.)
+    C++ example programs can be found in the examples folder of your
+    installation.
+    """
+
+    _name = "aocl-da"
+    homepage = "https://www.amd.com/en/developer/aocl/analytics.html"
```

Comment:
I cannot confirm the homepage.  Get a `404` error.

Reply:
Thank you for bringing this issue to our attention. The homepage URL has been updated correctly.

Reply:
Confirming, the new homepage URL works!